src/error.rs

```diff
@@ -66,13 +66,13 @@ impl ResponseError for TimeError {
             | TimeError::InvalidLength(_)
             | TimeError::BadId
             | TimeError::BadLeaderboardName => StatusCode::BAD_REQUEST,
-            TimeError::CurrentUser
-            | TimeError::UserExists
-            | TimeError::AlreadyFriends
+            TimeError::CurrentUser | TimeError::NotMember | TimeError::LastAdmin => {
```

Comment:
Does this need to be a block?

Reply:
cargo fmt turned it into a one so i guess it has to be

Reply:
so be it then